kolibri/plugins/management/assets/src/views/manage-content-page/channels-grid.vue

```diff
@@ -105,6 +112,11 @@
       this.currentTime = now();
       this.addChannelFileSummaries(map(this.channelList, 'id'));
     },
+    watch: {
+      channelList(val, newVal) {
```

Comment:
This isn't a critical issue, but my intuition after working on the manage-content-page for awhile is that the polling process could use a little optimization to not do unnecessary work, and clean up the vuex devtools.